.github/workflows/fronts.yml

```diff
@@ -0,0 +1,34 @@
+name: DCR Fronts Ophan tag status
+on:
+  # Triggers the workflow on push and pull request events but only for the main branch
+  push:
+    branches: [main]
+
+  # Allows you to run this workflow manually from the Actions tab.
+  workflow_dispatch:
+
+jobs:
+  data-link-name:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup deno
+        uses: denolib/setup-deno@v2
+        with:
+          deno-version: v1.21.0
+
+      - name: Run Deno script
+        id: text
+        run: |
+          echo "About to run Deno script"
```

Comment:
I'm concerned about running scripts outside of this repo. It'd be good to get this in here, I think it's just a matter of copying this script into this PR.

Reply:
I’m bringing it in!